sql-plugin/src/main/scala/org/apache/spark/sql/catalyst/json/rapids/GpuJsonScan.scala

```diff
@@ -138,6 +140,11 @@ object GpuJsonScan {
       meta.willNotWorkOnGpu("GpuJsonScan only supports \"\\n\" as a line separator")
     }
 
+    parsedOptions.encoding.foreach(enc =>
```

Comment:
nit: could we use `exists` here instead since this is an `Option`? Something like:

```scala
  if (parsedOptions.encoding.exists(enc =>
      enc != StandardCharsets.UTF_8.name() && enc != StandardCharsets.US_ASCII.name())) {
```

Reply:
Thx for the comment, I'd like to create a follow-up targeting 22.04